src/test/test-binfmt-util.c

```diff
@@ -0,0 +1,17 @@
+#include "binfmt-util.h"
+#include "tests.h"
+
+TEST(disable_binfmt) {
+        ASSERT_OK(disable_binfmt());
```

Comment:
Hmmm, `disable_binfmt()` breaks the current system config, hence it should not be called in unit test... Please drop the test case.